pintc/tests/unions/simple.pnt

```diff
@@ -109,8 +109,8 @@ predicate test {
 // predicate ::test {
 //     var ::x: ::thing;
 //     var ::d: int;
-//     var ::no_base_addr: ::maybe_addr;
-//     var ::a_base_addr: ::maybe_addr;
+//     var ::no_base_addr: int;
+//     var ::a_base_addr: int;
```

Comment:
This is not right -- two `var`s which were unions are now integers.  I'll look to fix it in my current branch.